src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -2284,52 +2471,116 @@ public override bool Equals(object obj)
+});
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)]
-        public async Task TestMissingReferences()
+        public async Task TestMissingReferences1()
         {
-            await TestWithPickMembersDialogAsync(
-@"
-<Workspace>
-    <Project Language='C#' AssemblyName='CSharpAssembly1' CommonReferences='false' LanguageVersion='CSharp6'>
-        <Document FilePath='Test1.cs'>
-public class Class1
+            await new Test
+            {
+                LanguageVersion = LanguageVersion.CSharp6,
+                CodeActionIndex = 1,
+                TestState =
+                {
+                    Sources =
+                    {
+@"public class Class1
 {
-    int i;
-    [||]
+    [|int i;|]
 
     public void F()
     {
     }
-}
-        </Document>
-    </Project>
-</Workspace>",
-@"
-public class Class1
+}",
+                    },
+                    ExpectedDiagnostics =
+                    {
+    // /0/Test0.cs(1,14): error CS0518: Predefined type 'System.Object' is not defined or imported
+    DiagnosticResult.CompilerError("CS0518").WithSpan(1, 14, 1, 20).WithArguments("System.Object"),
+    // /0/Test0.cs(1,14): error CS1729: 'object' does not contain a constructor that takes 0 arguments
+    DiagnosticResult.CompilerError("CS1729").WithSpan(1, 14, 1, 20).WithArguments("object", "0"),
+    // /0/Test0.cs(3,5): error CS0518: Predefined type 'System.Int32' is not defined or imported
+    DiagnosticResult.CompilerError("CS0518").WithSpan(3, 5, 3, 8).WithArguments("System.Int32"),
+    // /0/Test0.cs(5,12): error CS0518: Predefined type 'System.Void' is not defined or imported
+    DiagnosticResult.CompilerError("CS0518").WithSpan(5, 12, 5, 16).WithArguments("System.Void"),
+                    },
+                },
+                FixedState =
+                {
+                    Sources = {
+@"public class Class1
 {
     int i;
 
-    public override global::System.Boolean Equals(global::System.Object obj)
+    public override System.Boolean Equals(System.Object obj)
     {
         Class1 @class = obj as Class1;
-        return @class != null;
+        return @class != null &&
+               i == @class.i;
     }
 
     public void F()
     {
     }
 
-    public override global::System.Int32 GetHashCode()
-    {
-        return 0;
-    }
-}
-        ",
-chosenSymbols: new string[] { },
-index: 1);
+    public override System.Int32 GetHashCode()
+    {
+        return 165851236 + EqualityComparer<System.Int32>.Default.GetHashCode(i);
+    }
+}",
+                    },
+                    ExpectedDiagnostics =
+                    {
+    // /0/Test0.cs(1,14): error CS0518: Predefined type 'System.Object' is not defined or imported
+    DiagnosticResult.CompilerError("CS0518").WithSpan(1, 14, 1, 20).WithArguments("System.Object"),
```

Comment:
expected diganostics.  this tests us when we don't have a ref to mscorlib.  so there are tons of missing symbol errors (but no crashes).